13/umbraco-cms/fundamentals/setup/server-setup/iis.md

```diff
@@ -6,7 +6,7 @@ description: Information on hosting Umbraco on IIS
 
 ## Configuring IIS for .NET
 
-* Install the [ASP.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) and download the **Hosting Bundle**.
+* Install the [ASP.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/) and download the **Hosting Bundle**. Ensure you download the correct version for your specific version of Umbraco.
```

Comment:
ASP.NET is from below v9, and from v9 is just .NET. We also have the correct .NET version that needs to be installed in the requirements.
```suggestion
* Install the [NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/) and download the **Hosting Bundle**. Ensure you download the correct .NET version as per the [requirements](../requirements.md).
```